src/grafana_dashboards/MongoDB_ReplSet_Summary.json

```diff
@@ -2923,7 +2926,7 @@
                     "uid": "${prometheusds}"
                 },
                 "definition": "label_values({__name__=~\"pg_up|mysql_up|mongodb_up|proxysql_mysql_status_active_transactions\"}, replication_set)",
-                "hide": 2,
+                "hide": 0,
```

Comment:
is this needed

Reply:
Yes, it allows to select the replicat_set value